pom.xml

```diff
@@ -46,6 +50,19 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
```

Comment:
Are these new dependencies used by all the modules or just subset of modules?

Reply:
Just a subset. This is just to have the ability of sharing common logic across modules.

Reply:
I'm not really in favor of sharing common stuff across modules.

At this stage I still hesitate to introduce such module. I still believe in having all the things inside the module for easy extraction + keeping just the required deps per the module.

My experience with shared / common modules is not great, people tend to drop more and more things there and over time it becomes hell to maintain it and keep stuff in reasonable boundaries.

Reply:
I see your point. We could remove this common utils and have an infinispan module for the test resource only, but taking into account that there are only two modules that would use it, it does not bring up a lot of value. 
Will remove this module and have the infinispan test resource duplicated.

Reply:
One more thing, people need to run `mvn test -pl 700-quarkus-infinispan-grpc-kafka,common-test-utils` instead of `mvn test -pl 700-quarkus-infinispan-grpc-kafka` and it is easy to forget about it / use old common-test-utils installed in local repo

Reply:
Removed